go.mod

```diff
@@ -47,7 +47,7 @@ replace (
 )
 
 // maistra forks
-replace sigs.k8s.io/controller-runtime => github.com/maistra/controller-runtime v0.1.13-0.20190926110539-8c6fdfaa0ad1
+replace sigs.k8s.io/controller-runtime => github.com/maistra/controller-runtime v0.1.13-0.20191029133215-623f07679978
```

Comment:
That's odd. The commit's timestamp is 13:32:15:
```
commit 623f07679978e887bb5bf42e4c647dd5173924c1 (HEAD -> maistra-1.1, origin/maistra-1.1, origin/HEAD)
Author: Marko Lukša <marko.luksa@gmail.com>
Date:   Tue Oct 29 13:32:15 2019 +0100
```

I wonder where it's getting the 12:44:51 timestamp. What go version are you using?

Reply:
Oh, I see: 
```
git log --pretty=fuller
commit 623f07679978e887bb5bf42e4c647dd5173924c1 (HEAD -> maistra-1.1, origin/maistra-1.1, origin/HEAD)
Author:     Marko Lukša <marko.luksa@gmail.com>
AuthorDate: Tue Oct 29 13:32:15 2019 +0100
Commit:     Marko Lukša <marko.luksa@gmail.com>
CommitDate: Tue Oct 29 13:44:51 2019 +0100
```

Definitely caused by a different go version.

Reply:
Interesting. I'm on 1.13.3

Reply:
I was using 1.12, but have since tried 1.13.5 and 1.13.3 just now. Neither of them complained about the incorrect pseudo version, so I'm not sure what's going on.

Reply:
Hmm. They introduced version validation in 1.13, awkward you didn't see it: https://golang.org/doc/go1.13#version-validation

Reply:
Looks like it doesn't validate it any more, since I already have it cached locally. I've now changed it to the correct timestamp. Thanks!